engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1192,8 +1192,12 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                     handlePath(vmTO.getDisks(), vm.getHypervisorType());
 
                     Commands cmds = new Commands(Command.OnError.Stop);
+                    final Map<String, String> sshAccessDetails = _networkMgr.getSystemVMAccessDetails(vm);
```

Comment:
Why not wrapping these on an if block? (when VM type = systemVM)

Reply:
these values however get consumed in a block that's executed only for CPVM & SSVM